force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Security (ISVs): Missing or incorrect sharing clause
#### Severity
Error

#### Finding
Class `UtilityApexHelper` does not enforce sharing correctly. Please change to ``Inherited Sharing``.

#### Why is this a problem?
Because Apex generally runs in system context, permissions, field-level security, and sharing rules aren't taken into account during code execution. This might put applications at risk of inadvertently exposing sensitive data.

#### Help and documentation

* [Use Inherited Sharing to Secure Your Apex Code (Salesforce Release Notes)](https://releasenotes.docs.salesforce.com/en-us/winter19/release-notes/rn\_apex\_inherited\_sharing.htm) 
* [Violation of Secure Design Principles (CWE)](https://cwe.mitre.org/data/definitions/657.html) 
* [A6-Security Misconfiguration (OWASP)](https://www.owasp.org/index.php/Top\_10-2017\_A6-Security\_Misconfiguration)